qiskit_nature/drivers/second_quantization/pyscfd/pyscfdriver.py

```diff
@@ -209,6 +200,26 @@ def spin(self, spin: int) -> None:
         """Sets the spin."""
         self._spin = spin
 
+    @property
+    def basis(self) -> str:
+        """return basis"""
+        return self._basis
+
+    @basis.setter
+    def basis(self, value: str) -> None:
+        """set basis"""
+        self._basis = value
+
+    @property
+    def method(self) -> MethodType:
+        """return Hartree-Fock method"""
```

Comment:
For PySCF it now supports DFT methods too. We can shorten it to say `Returns method` or have it be a be more than stating what the code already tells you by saying `Returns Hartree-Fock/Kohn-Sham method`  @mrossinek any thought

Reply:
I added the typehint return and completed the docstrings with the exception

Reply:
Yes that is true. I think the suggested docstring should be fine :+1:

Reply:
Changed the docstring